products/sle/main.pm

```diff
@@ -464,25 +464,30 @@ sub load_slenkins_tests {
     return 0;
 }
 
-sub load_cluster_boot {
-    if (check_var('ARCH', 'aarch64')) {
-        # We need to boot *before* waiting for support server
-        # Because of TianoCore UEFI boot method
-        loadtest "boot/boot_to_desktop";
-        # Wait for support server to complete its initialization
-        loadtest "support_server/wait_support_server";
-        # TODO: poo#32110 add reboot or ensure that network is up
+sub boot_hpc {
+    if (get_var('CLUSTER_NODES')) {
+        if (check_var('ARCH', 'aarch64')) {
+            # We need to boot *before* waiting for support server
+            # Because of TianoCore UEFI boot method
+            loadtest "boot/boot_to_desktop";
+            # Wait for support server to complete its initialization
+            loadtest "support_server/wait_support_server";
+            # TODO: poo#32110 add reboot or ensure that network is up
+        }
+        else {
+            # Wait for support server to complete its initialization
+            loadtest "support_server/wait_support_server";
+            loadtest "boot/boot_to_desktop";
+        }
+        # Standard boot and configuration
+        loadtest "qa_automation/patch_and_reboot" if is_updates_tests;
+        loadtest "console/consoletest_setup"      if is_updates_tests;
     }
     else {
-        # Wait for support server to complete its initialization
-        loadtest "support_server/wait_support_server";
-        loadtest "boot/boot_to_desktop";
+        loadtest 'boot/boot_to_desktop';
     }
-    # Standard boot and configuration
-    loadtest "qa_automation/patch_and_reboot" if is_updates_tests;
-    loadtest "console/consoletest_setup"      if is_updates_tests;
     loadtest "console/hostname";
-
+    loadtest 'hpc/before_test' if (sle_version_at_least('15'));
```

Comment:
In SLE15 currently it's not an issue , but I am not sure about QAM , @czerw  ?

Reply:
... for versions older SLE15 :)

Reply:
This will not work for QAM  when is SLE15 moved to maintenance. Could  you please and "AND" condition and check existence of variable HPC_REPO? This variable is not used by QAM test.